index.html

```diff
@@ -4201,6 +4207,458 @@ <h3>Device Theft and Impersonation</h3>
     </section>
   </section>
 
+  <section class="appendix">
+    <h2>Proof Format Trade-Offs</h2>
+
+    <p>
+The Verifiable Credentials Data Model is designed to be compatible with a
+variety of existing and emerging syntaxes and digital proof formats. Each
+format has benefits and drawbacks. The following table is intended to summarize
+a number of these native trade-offs.
+    </p>
+
+    <p>
+The table below compares three proof format ecosystems; JSON+JWTs,
+JSON-LD+JWTs, and JSON-LD+LD-Proofs. At the time of publication of this
+specification, no organization is planning to use JSON+JWTs. The existence
```

Comment:
This is incorrect. Our base layer at uPort is JSON+JWT's with JSON-LD an option for implementers.

Reply:
Ok, thanks. Fixed in 0691a7e6f3adb294fa914e22c2fde56504bcf72d.